### PR TITLE
created notification app and fixed all imports

### LINKED ISSUE #1029: Create `notifications` app with initial code
### Idea summary

Create a new `notifications` app. Move related code from the `api` app to `notifications`

### Further details

Steps to complete this task:

- [ ] create a notifications app
	- `python manage.py startapp notifications`
- [ ] add the `notifications` app under `INSTALLED_APPS` in `core/settings.py` 
- [ ] move notification-related code from the `api` to the `notifications` app
    - [ ] `api/models/notification.py` -> `notifications/models.py`
    - [ ] remove [api/__init__.py notification import](https://github.com/CiviWiki/OpenCiviWiki/blob/882f9fd522830f5849eddbc9802e4a6ed24d29d3/project/api/models/__init__.py#L8)
- [ ] create database migration(s)
    - [ ] `python manage.py makemigrations`
    - [ ] `python manage.py migrate`
- [ ] Commit changes, push to GitHub, and open a pull request against the `main` branch in this repository

### PR DESCRIPTION
**Pull request by brylie**
@anurag629's work rebased against `develop`.

Closes #1029

### CODE REVIEW

**Comment by brylie** [comment]
@gorkemarslan would you please review this pull request?

File: project/api/models/notification.py [hunk 1]
```diff
@@ -1,36 +0,0 @@
...
-        ("response_to_yout_civi", "Response to your civi"),
-        ("rebuttal_to_your_response", "Rebuttal to your response"),
-    )
-    activity_type = models.CharField(
-        max_length=31, default="new_follower", choices=activity_CHOICES
-    )
-    read = models.BooleanField(default=False)
-
-    created = models.DateTimeField(auto_now_add=True, blank=True, null=True)
-    last_modified = models.DateTimeField(auto_now=True, blank=True, null=True)
```
Comment (gorkemarslan) [inline project/api/models/notification.py]: @anurag629 and @brylie,
Now, `project/api/models/notification.py` is completely empty so it's better to delete this file.

**Review by gorkemarslan — CHANGES_REQUESTED** [review]

File: project/api/models/notification.py (hunk 1 above)
Comment (brylie) [reply project/api/models/notification.py]: Done

**Review by gorkemarslan — APPROVED** [review]

**Comment by brylie** [comment]
@all-contributors please add @anurag629 for code

**Pull request merged**